src/main/java/yevhenii/lostfilmdemo/repository/UserRequestRepository.java

```diff
@@ -0,0 +1,20 @@
+package yevhenii.lostfilmdemo.repository;
+
+import org.springframework.transaction.annotation.Transactional;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+
+import java.util.List;
+import java.util.Optional;
+
+@Transactional
+public interface UserRequestRepository {
```

Comment:
I understood the purpose of UserRequestRepository and SchedulerRequestRepository